SVProgressHUD/SVProgressHUD.m

```diff
@@ -1352,7 +1352,11 @@ - (CGFloat)visibleKeyboardHeight {
                 for (__strong UIView *possibleKeyboardSubview in possibleKeyboard.subviews) {
                     viewName = NSStringFromClass(possibleKeyboardSubview.class);
                     if([viewName hasPrefix:@"UI"] && [viewName hasSuffix:@"InputSetHostView"]) {
-                        return CGRectGetHeight(possibleKeyboardSubview.bounds);
+                        CGRect convertedRect = [possibleKeyboard convertRect:possibleKeyboardSubview.frame toView:self];
+                        CGRect intersectedRect = CGRectIntersection(convertedRect, self.bounds);
+                        if (!CGRectIsNull(intersectedRect)) {
```

Comment:
and what is  CGRectIsNull is return null

Reply:
@Sumit-2singh 

There is 
```swift
return 0
```

at the end of the method (line [1366](https://github.com/SVProgressHUD/SVProgressHUD/pull/846/files/981c7b58973e4b3382395ffbd701989207c77959#diff-a9f291dcb26df32018013622ccd7783dR1366)), so it'll return `0`.

Reply:
@Sumit-2singh 

There is 
```objc
return 0;
```

at the end of the method (line [1366](https://github.com/SVProgressHUD/SVProgressHUD/pull/846/files/981c7b58973e4b3382395ffbd701989207c77959#diff-a9f291dcb26df32018013622ccd7783dR1366)), so it'll return `0`.